readme.md

```diff
@@ -47,8 +47,9 @@ or
 
 PoExtractor assumes, the code follows several conventions:
 
-* `IStringLocalizer` or a derived class is accessed via a property named `T`, `S`, `TS`, `H` or `TH`
-* Liquid templates use the filter named `t`
+* `IStringLocalizer` or a derived class is accessed via a field named `S` (This is a convention used in Orchard Core)
```

Comment:
I'm not sure if these are limitations or conventions used by Orchard Core & Fluid, even though we can make an extensibility point to change them if it's required